src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0018Tests.cs

```diff
@@ -193,29 +215,28 @@ await Verifier.CreateAnalyzer(code)
         }
 
         [Fact]
-        public async Task AZC0018ProducedForMethodsWithParameterModel()
+        public async Task AZC0018ProducedForMethodsWithOperationOfPageableModel()
         {
             const string code = @"
 using Azure;
 using Azure.Core;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Collections.Generic;
 
 namespace RandomNamespace
 {
-    public struct Model
```

Comment:
Same comment about keeping the old test and adding a new one.

Reply:
Same above. I think I'm not deleting.